vault/identity_store.go

```diff
@@ -223,20 +236,37 @@ func mfaPaths(i *IdentityStore) []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ReadOperation: &framework.PathOperation{
 					Callback: i.handleMFAMethodTOTPRead,
-					Summary:  "Read the current configuration for the given MFA method",
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb:   "read",
+						OperationSuffix: "totp-method-configuration|totp-method-configuration",
+					},
+					Summary: "Read the current configuration for the given MFA method",
 				},
 				logical.UpdateOperation: &framework.PathOperation{
 					Callback: i.handleMFAMethodTOTPUpdate,
-					Summary:  "Update or create a configuration for the given MFA method",
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb:   "configure",
+						OperationSuffix: "totp-method|totp-method",
+					},
+					Summary: "Update or create a configuration for the given MFA method",
 				},
 				logical.DeleteOperation: &framework.PathOperation{
 					Callback: i.handleMFAMethodTOTPDelete,
-					Summary:  "Delete a configuration for the given MFA method",
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb:   "delete",
+						OperationSuffix: "totp-method|totp-method",
```

Comment:
should this be `totp-method-configuration` so it becomes `delete-totp-method-configuration`

Reply:
I was following the documentation on these (https://developer.hashicorp.com/vault/api-docs/secret/identity/mfa/totp#delete-totp-mfa-method). If I understand correctly, this deletes the entire TOTP MFA method, not just the configuration.